.github/workflows/ci.yml

```diff
@@ -67,14 +73,21 @@ jobs:
 
       - name: Install Python packages
         run: |
-          pip install --upgrade codecov tox setuptools
+          pip install --upgrade codecov tox
           pip list
 
       - name: Display tool versions
         run: python misc/build_helpers/show-tool-versions.py
 
-      - name: Run "tox -e py27-coverage"
+      - name: Run coverage tests (Python 2.7)
```

Comment:
They're not really coverage tests.  They don't test anything about coverage.  They are just the tests, with coverage measurement turned on.